File: OrderManagment/Dto/ViewOrder.cs
using OrderManagment.Models;

namespace OrderManagment.Dto
{
    public class ViewOrder
    {
        public int Id { get; set; }
        public DateTime CreatedAt { get; set; }
        public Contractor Contractor { get; set; }
        public double OverallNetto { get; set; }
        public double OverallBrutto { get; set; }
        public List<OrderProducts> OrderedProducts { get; set; }
    }
}

File: OrderManagmentUI/src/app/models/ViewOrder.ts
import { ViewOrderProductsComponent } from "../components/view-order-products/view-order-products.component";
import { ViewContractor } from "./ViewContractor";

export class ViewOrder{
    id?: number;
    date ="";
    contractor: ViewContractor=new ViewContractor();
    overallNetto?: number;
    overallBrutto?: number;
    orderedProducts:ViewOrderProductsComponent[]=[];  
}